www/src/pages/en/mutations/transact-write.mdx

```diff
@@ -0,0 +1,324 @@
+---
+title: Transact Write
+description: Performing write transactions
+keywords:
+  - electrodb
+  - docs
+  - dynamodb
+  - TransactWrite
+  - transactWrite
+  - transaction
+  - transactions
+layout: ../../../layouts/MainLayout.astro
+---
+
+import TableDefinition from '../../../partials/table-definition.mdx';
+
+<TableDefinition />
+
+In cases where you must keep multiple records in sync, enforce constraints across entities, and/or ensure request idempotency you can use ElectroDB's Transact [Get](/en/queries/transact-get)/[Write](/en/mutations/transact-write) methods. There are many articles and guides on use-cases for DynamoDB's Transaction APIs, resource links can be found [at the bottom of this document](#resources). This page will focus on how to use ElectroDB's transaction API.
+
+## Performing Write Transactions
+
+To perform `write` transactions with ElectroDB you will first need to create a [Service](/en/modeling/services). Available on the `Service` exist two methods `transaction.get()` and `transaction.write()`. These methods accept a callback function which is then provided with the entities of the service, and should return an array of mutations. Creating mutations within a transaction is nearly identical to other mutations, only instead of terminating your query with a `.go()` or `.params()` you use `.commit()` instead.
```

Comment:
```suggestion
To perform `write` transactions with ElectroDB you will first need to create a [Service](/en/modeling/services). Available on the `Service` exist two methods: `transaction.get()` and `transaction.write()`. These methods accept a callback function, which is then provided with the entities of the service, should return an array of mutations. Creating mutations within a transaction is nearly identical to other mutations except instead of terminating your query with `.go()` or `.params()` you use `.commit()` instead.
```